assets/scss/_custom.scss

```diff
@@ -639,3 +639,10 @@ body.td-documentation {
   }
 }
 
+.td-content {
+  table code {
+    background-color: inherit !important;
+    color: inherit !important;
+    font-size: inherit !important;
```

Comment:
If we can use a more specific selector to avoid using `!important`, that's better for accessibility.